usage/bigquery.go

```diff
@@ -17,21 +17,28 @@ type QueryFilter struct {
 }
 
 type QuerySpec struct {
+	TimeStep string
 	From, To *time.Time
 	Filter   QueryFilter
 }
 
+var allowedTimeSteps = map[string]bool{
+	"hour": true,
+	"day":  true,
+}
+
 type UsageSummaryRow struct {
 	UserID    string `bigquery:"user_id"`
 	CreatorID string `bigquery:"creator_id"`
 
-	ViewCount       int64              `bigquery:"view_count"`
-	LegacyViewCount bigquery.NullInt64 `bigquery:"legacy_view_count"`
-	PlaytimeMins    float64            `bigquery:"playtime_mins"`
+	DeliveryUsageGbs float64 `bigquery:"delivery_usage_gbs"`
```

Comment:
Wasn't delivery gonna be in minutes as well? Maybe weird that this is the only one in GB making the metrics not comparable with each other (e.g. how bigger is delivery vs storage)

Reply:
Yeah but the bigquery column changed only lately! Just pushed the commit addressing this